### PR TITLE
feat: add Watch section support

### PR DESCRIPTION
**Pull request by pospisilf**
Before submitting your PR, please review the following checklist:

- [ ] **CONSIDER** adding a new test if your PR resolves an issue.
- [ ] **DO** keep pull requests small so they can be easily reviewed.
- [ ] **DO** make sure tests pass.
- [ ] **DO** make sure any public APIs changes are documented.
- [ ] **DO** make sure not to introduce any compiler warnings.

Before merging the PR:

- [ ] **CHECK** continuous integration of `main` branch is green.
- [ ] **CHECK** pull request check job is green.
- [ ] **CHECK** all pull request questions/requests are resolved.
- [ ] **WAIT** till PR is approved by at least 1 committer.

### CODE REVIEW

File: packages/page-objects/src/components/sidebar/tree/debug/WatchSection.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	constructor(panel: WebElement, viewContent: ViewContent) {
+		super(panel, viewContent, WatchSectionItem);
+	}
+
+	/**
+	 * Add item to Watch section.
+	 * @param name
+	 */
+	async addItem(name: string): Promise<void> {
+		await (await this.getAction('Add Expression'))?.click();
```
Comment (djelinek) [inline packages/page-objects/src/components/sidebar/tree/debug/WatchSection.ts:33]: lets make from all button labels the locators please

File: packages/page-objects/src/components/sidebar/tree/debug/WatchSection.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		await new Promise((res) => setTimeout(res, 1000));
+		const textInput = await this.findElement(By.css('input'));
```
Comment (djelinek) [inline packages/page-objects/src/components/sidebar/tree/debug/WatchSection.ts:35]: lets move `By.css` into locators

File: tests/test-project/src/test/debug/debug.test.ts [hunk 3]
```diff
@@ -39,7 +40,7 @@ import { expect } from 'chai';
 const line = 7;
 const varSub = VSBrowser.instance.version >= '1.91.0' ? 'num =' : 'num:';
 
-describe('Debugging', function () {
+describe.only('Debugging', function () {
```
Comment (djelinek) [inline tests/test-project/src/test/debug/debug.test.ts:43]: do not forget to remove `only`

File: tests/test-project/src/test/debug/debug.test.ts [hunk 4]
```diff
@@ -291,6 +293,56 @@ describe('Debugging', function () {
 			await item?.setVariableValue('42');
 			item = await getNumVariable(view, this.timeout() - 2000);
 			expect(await item?.getVariableValue()).equals('42');
+			await VSBrowser.instance.driver.actions().sendKeys(Key.ESCAPE).perform();
```
Comment (djelinek) [inline tests/test-project/src/test/debug/debug.test.ts:296]: why there is a need of `ESC` and static sleep?

File: docs/DebugView.md [hunk 5]
```diff
@@ -25,3 +25,55 @@ await debugConfiguration.selectLaunchConfiguration("Test Launch");
 // start selected launch configuration
 await debugView.start();
 ```
+
+### Sections
```
Comment (djelinek) [inline docs/DebugView.md:29]: I like that you have added all supported sections . 🚀 

Could we please also add detailed screenshots for each section with some items there? to better showcase the usecase? 🙂

**Review by djelinek — DISMISSED** [review]

File: docs/DebugView.md (hunk 5 above)
Comment (pospisilf) [reply docs/DebugView.md:29]: added

**Review by djelinek — APPROVED** [review]

**Pull request merged**